Dockerfile

```diff
@@ -2,6 +2,12 @@ FROM rocker/shiny
 
 MAINTAINER "Nicholas Heyek and David LeBauer"
 
+ENV bety_dbname=${bety_dbname:-bety}
```

Comment:
just use ENV bety_dbname=bety. This is used at compile time. When you want to modify it when running you can use `docker run -e bety_dbname=foobar`